godot-macros/src/class/godot_api.rs

```diff
@@ -257,9 +296,14 @@ fn process_godot_fns(decl: &mut Impl) -> Result<(Vec<FuncDefinition>, Vec<Functi
                     if method.return_ty.is_some() {
                         return attr.bail("return types are not supported", method);
                     }
+                    let attributes = method.attributes.clone();
                     let sig = util::reduce_to_signature(method);
 
-                    signal_signatures.push(sig.clone());
+                    signal_definitions.push(SignalDefinition {
+                        // TODO: why is this cloned?
+                        signature: sig.clone(),
+                        attributes,
+                    });
```

Comment:
It seems to me that we can remove the `.clone()` from `sig`, since `reduce_to_signature` already clones `method`, so we'd be cloning twice. Thoughts?

Reply:
Okay, done!